spec/javascripts/govuk_publishing_components/load-analytics.spec.js

```diff
@@ -0,0 +1,323 @@
+/* eslint-env jasmine */
+
+var linkedDomains = [
```

Comment:
Do we need all of these linkedDomains listed in the test? Is there not a way to fake them in the test for comparison purposes?

Reply:
@andysellick I'd need to move the `linkedDomains` variable outside of `loadUa()` and directly onto `window.GOVUK.loadAnalytics` so that I can modify it in the test. I'll do that.